src/main/java/com/untamedears/ItemExchange/utility/ExchangeRule.java

```diff
@@ -545,47 +548,79 @@ public int checkMultiples(Inventory inventory) {
 	 * Checks if the given ItemStack follows the ItemRules except for the amount
 	 */
 	public boolean followsRules(ItemStack itemStack) {
-		// check material type and druability
-		boolean followsRules = DeprecatedMethods.getMaterialId(material) == DeprecatedMethods.getItemId(itemStack)
-				&& durability == itemStack.getDurability();
-		
-		// Check enchantments
-		if (itemStack.getEnchantments().size() > 0) {
-			followsRules = followsRules && itemStack.getEnchantments().entrySet().containsAll(requiredEnchantments.entrySet());
-			for (Enchantment excludedEnchantment : excludedEnchantments) {
-				followsRules = followsRules && !itemStack.getEnchantments().entrySet().contains(excludedEnchantment);
+		// If not the same material, return false
+		if (DeprecatedMethods.getMaterialId(material) != DeprecatedMethods.getItemId(itemStack)) {
```

Comment:
Could have just done `getType` on the ItemStack and then compare materials via `==`, no ?

Reply:
Yes actually, my bad